internal/app/subsystems/aio/store/sqlite/sqlite.go

```diff
@@ -232,7 +279,7 @@ type SqliteStoreWorker struct {
 }
 
 func New(config *Config) (aio.Subsystem, error) {
-	db, err := sql.Open("sqlite3", config.Path)
+	db, err := sql.Open("sqlite3", config.Path+"?_foreign_keys=on")
```

Comment:
Do we still need the foreign_keys setting?

Reply:
not anymore. thanks